templates/home.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block title %} Side Hustle{% endblock %}
+
+{% block content %}
+
+<h1>Welcome to SideHustle</h1>
+
+
+{% endblock %}
```

Comment:
nit: missing newline here

Reply:
The error message was deleted.
This is not relevant to the flake8 check.